ui/component/publish/livestream/livestreamForm/view.jsx

```diff
@@ -0,0 +1,564 @@
+  }, [mode, _uploadType]);
+
+  async function handlePublish() {
+    let outputFile = filePath;
+    let runPublish = false;
+
+    // Publish file
+    if (mode === PUBLISH_MODES.FILE || isLivestreamMode) {
+      runPublish = true;
+    }
+
+    if (runPublish) {
+      if (enablePublishPreview) {
+        setPreviewing(true);
+        publish(outputFile, true);
+      } else {
+        publish(outputFile, false);
+      }
+    }
+  }
+
+  // When accessing to publishing, make sure to reset file input attributes
+  // since we can't restore from previous user selection (like we do
+  // with other properties such as name, title, etc.) for security reasons.
+  useEffect(() => {
+    if (mode === PUBLISH_MODES.FILE) {
+      updatePublishForm({ filePath: '', fileDur: 0, fileSize: 0 });
+    }
+  }, [mode, updatePublishForm]);
+
+  // File Source Selector State.
+  const [fileSource, setFileSource] = useState();
+  const changeFileSource = (state) => setFileSource(state);
+
+  const [showSchedulingOptions, setShowSchedulingOptions] = useState(false);
+  useEffect(() => {
+    setShowSchedulingOptions(isLivestreamMode && fileSource === SOURCE_NONE);
+  }, [isLivestreamMode, fileSource]);
+
+  if (publishing) {
+    return (
+      <div className="main--empty">
+        <h1 className="section__subtitle">{__('Publishing...')}</h1>
+        <Spinner delayed />
+      </div>
+    );
+  }
+
+  const isFormIncomplete =
+    isClaimingInitialRewards ||
+    formDisabled ||
+    !formValid ||
+    uploadThumbnailStatus === THUMBNAIL_STATUSES.IN_PROGRESS ||
+    ytSignupPending ||
+    previewing;
+
+  // Editing claim uri
+  return (
+    <div className="card-stack">
+      <Card className="card--livestream">
+        <div>
+          <Button
+            key={'New'}
+            icon={ICONS.LIVESTREAM}
+            iconSize={18}
+            label={'New Livestream'}
+            button="alt"
+            onClick={() => {
+              // $FlowFixMe
+              setPublishMode('New');
+            }}
+            className={classnames('button-toggle', { 'button-toggle--active': publishMode === 'New' })}
+          />
+          <Button
+            key={'Replay'}
+            icon={ICONS.MENU}
+            iconSize={18}
+            label={'Choose Replay'}
+            button="alt"
+            onClick={() => {
+              // $FlowFixMe
+              setPublishMode('Replay');
+            }}
+            disabled={!hasLivestreamData}
+            className={classnames('button-toggle', { 'button-toggle--active': publishMode === 'Replay' })}
+          />
+        </div>
+        {!isMobile && <ChannelSelect hideAnon autoSet channelToSet={claimChannelId} isTabHeader />}
+        <Button
+          button="secondary"
+          label={__('Check for Replays')}
+          disabled={isCheckingLivestreams}
+          icon={ICONS.REFRESH}
+          onClick={() => fetchLivestreams(claimChannelId, activeChannelName)}
+        />
+      </Card>
```

Comment:
- [ ] Not sure why the tooltop is not appearing.  Didn't know what the button was for.
- [ ] If you use the keyboard to select a channel, there is an odd border, plus my membership icon becomes white-ish.

<img width="228" alt="image" src="https://user-images.githubusercontent.com/64950861/176882696-0ecbcfd7-21df-4c15-8348-72ec7107b75c.png">